### ISSUE TITLE
Outline and refine the requirements for the 1.2 release

### ISSUE THREAD

**Issue by iirelu** [mypaint/mypaint#147]
[Looking at the list of things under the 1.2 milestone](https://github.com/mypaint/mypaint/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22MyPaint+1.2.0%22), there's a large amount of stuff that isn't important enough to hold up a release for. It's almost time to start thinking about pre-release versions, so having a definitive list of goals for 1.2's release is important.

v1.1.0 was released December 31st 2012, which means there has not been an official release in nearly two years. We should start pruning enhancements from the 1.2 milestone so the amount of work needed for it becomes manageable.

Coordination about how to actually manage the release will likely be in another issue.

**Comment by achadwick** [comment]
One thing that might help us progress the next release is to make interim beta release/milestones which are smaller than the mass of things tagged as "1.2". Smaller task-lists are less off-putting, and it would be good to start prioritizing and triaging for the beta cycle now.

The versioning strategy I envisage for this is documented at [lib/meta.py:11](https://github.com/mypaint/mypaint/blob/ffacdd7960af37c8b0c3f4b4ea3659124fb06405/lib/meta.py#L11). Could you review and check that it's still sensible please?

**Comment by achadwick** [comment]
(Assuming it was ever sensible in the first place, of course /selfdeprec)

**Comment by achadwick** [comment]
New milestones:

* [1.2.0-alpha](https://github.com/mypaint/mypaint/milestones/MyPaint%201.2.0-alpha)
  - for all work which needs to be done before the beta cycle can start
  - this means string changes and new features, plus any important bugs
* [1.2.0-beta.<1st>](https://github.com/mypaint/mypaint/milestones/MyPaint%201.2.0-beta.%3C1st%3E)
  - first beta milestone (of possibly many)
  - formal beta release once this is complete, no date yet
  - bugfixes only, ideally trivial ones; no string changes without good reason, and no new features

Next task is to try and triage all the unclassified items from the [list of open 1.2.0 issues](https://github.com/mypaint/mypaint/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22MyPaint+1.2.0%22) and decide if they're going into the alpha cycle (strings, features, important bugs) or the beta cycle (minor bugfixes, organizing translations).

Once that's done, we should probably close down this project issue too...!

**Comment by achadwick** [comment]
I guess it's worth noting that simple artwork changes like #81 are good things to address in the beta cycle too. It's an enhancement, but not really a coding feature.

**Comment by iirelu** [comment]
Splitting things up into manageable tasks sounds like a good idea!

**Comment by achadwick** [comment]
I've renamed the "MyPaint 1.2.0" milestone to "[MyPaint 1.1.1-alpha.\<final\>](https://github.com/mypaint/mypaint/milestones/MyPaint%201.1.1-alpha.%3Cfinal%3E)", and I'm in the process of migrating stuff away from it to later milestones and triaging all the issues as I go. Expect it to be completed within the week, and when that's done we start the 1.2.0-alpha cycle!

I've also created a milestone for tracking those pesky floating project management tasks that will hold up the 1.2.0 release, but are unrelated to actual software development.
